.github/workflows/future-release-branch-labeler.yml

```diff
@@ -0,0 +1,34 @@
+name: Future Release Branch Pull Request Labeler
+
+on:
+  pull_request_target:
+    branches:
+      - dev-[0-9].[0-9]+
+    types:
+      - opened
+      - reopened
+
+jobs:
+  add-milestone:
+    runs-on: ubuntu-latest
+    steps:
+      - name:
+        run: |
+          # determine the target release from the branch that this PR targets
+          # ignore first 4 characters ("dev-")
+          milestone=${GITHUB_BASE_REF:4}
```

Comment:
What is the value of `milestone` at line 19?

Reply:
eg `1.19`

Reply:
Is there enough info to GET the milestone instead of the list of milestones?

Reply:
You can't call the GET endpoint for the milestone without the milestone number

Reply:
Yes. Sorry for so many questions. I am just trying to understand.

Reply:
It's not a problem! I think maybe a short demo would also help out here for everyone? I can prepare something short for the next sig docs meeting and answer any questions there as well

Reply:
Would this work?
--html_url 'https://api.github.com/repos/${{ github.repository }}/milestones/v$milestone

Reply:
I can try it out later today and will post an update here.

Reply:
I don't need a demo. More interested in how to use the API in the least number of steps.